qemu/qmp.go

```diff
@@ -938,12 +938,17 @@ func (q *QMP) ExecHotplugMemory(ctx context.Context, qomtype, id, mempath string
 }
 
 // ExecutePCIVSockAdd adds a vhost-vsock-pci bus
-func (q *QMP) ExecutePCIVSockAdd(ctx context.Context, id, guestCID, vhostfd string, disableModern bool) error {
+func (q *QMP) ExecutePCIVSockAdd(ctx context.Context, id, guestCID, vhostfd, addr, bus string, disableModern bool) error {
 	args := map[string]interface{}{
 		"driver":    VHostVSockPCI,
 		"id":        id,
 		"guest-cid": guestCID,
 		"vhostfd":   vhostfd,
+		"addr":      addr,
```

Comment:
Why are we enforcing `addr`, but leave `bus` as optional ?

Reply:
yup, same as `PCIVFIO` https://github.com/intel/govmm/blob/master/qemu/qmp.go#L837-L848

Reply:
This seems to contradict the commit message though

> addr and bus must be specified when a new device is hot plugged
on machines based on PCIe, like q35.

Reply:
but in the case of PCI (pc) the bus is optional, since it can be hot plugged directly on the root bus

Reply:
I see.  The problem is the commit message is very confusing.  It talks about making the bus parameter mandatory but if you look in the code that it adds the bus parameter appears to be optional.  It seems that both I and @sboeuf were confused by this so it's probab